packages/lesswrong/components/editor/EditorFormComponent.tsx

```diff
@@ -65,7 +65,11 @@ export const EditorFormComponent = ({form, formType, formProps, document, name,
   
   const defaultEditorType = getUserDefaultEditor(currentUser);
   const currentEditorType = contents.type || defaultEditorType;
-  const showEditorWarning = (formType !== "new") && (initialEditorType !== currentEditorType) && (currentEditorType !== 'ckEditorMarkup')
+
+  const showEditorWarning = (formType !== "new") 
+    && (defaultEditorType !== currentEditorType) // if the user has a default editor, and it's not the same as the current editor
+    && (currentEditorType !== 'ckEditorMarkup') // the editor warning button only lets you convert things back to ckEditorMarkup,
+    // so we don't bother showing it if you're already in ckEditorMarkup
```

Comment:
This is effectively a revert of https://github.com/ForumMagnum/ForumMagnum/commit/e3a20b073d73eebeb255c06862d29e0b7306a442, which looks correct to me.